File: integration-tests/mongodb-panache/src/main/java/io/quarkus/it/mongodb/panache/test/TestReactiveRepository.java
package io.quarkus.it.mongodb.panache.test;

import jakarta.enterprise.context.ApplicationScoped;

import io.quarkus.mongodb.panache.reactive.ReactivePanacheMongoRepository;

@ApplicationScoped
public class TestReactiveRepository implements ReactivePanacheMongoRepository<TestReactiveEntity> {
}

File: integration-tests/mongodb-panache-kotlin/src/main/kotlin/io/quarkus/it/mongodb/panache/test/TestReactiveRepository.kt
package io.quarkus.it.mongodb.panache.test

import io.quarkus.mongodb.panache.kotlin.reactive.ReactivePanacheMongoRepository
import jakarta.enterprise.context.ApplicationScoped

@ApplicationScoped
class TestReactiveRepository : ReactivePanacheMongoRepository<TestReactiveEntity>
